scripts/qa_crowbarsetup.sh

```diff
@@ -3261,23 +3228,15 @@ function set_proposalvars
         deployceph=
     fi
     # C3: Cloud5 only has ceph for SLES12
-    if iscloudver 5 && [ -z "$want_sles12" ] ; then
+    if iscloudver 5 ; then
```

Comment:
the job is disabled and has not passed a single time:

https://ci.suse.de/view/Cloud/view/Cloud6/job/cloud-mkcloud6-job-upgrade-x86_64/